src/service/synonymService.ts

```diff
@@ -14,9 +14,11 @@ if (process.env.STORAGE_LOCATION){
 
 
 const synonymService = {
-    getSynonym : async (from : string) : Promise<Synonym | undefined> => {
+    getSynonym : async (inputTerm : string) : Promise<Synonym | undefined> => {
         const allSynonyms = await synonymService.getAllSynonyms();
-        return allSynonyms.find(({from : savedFrom}) => savedFrom == from);
+        return allSynonyms.find(({from : savedFrom, target : savedTarget }) => 
```

Comment:
This is the other change I mentioned in the PR description, this allows searches in iPlayarr directly to respect the filename whether they enter the correct show name or the *arr show name. I also made it ignore case to increase the likelihood of matching, as I was confused why it wasn't applying and realised I hadn't title cased my search term. Happy to revert this if you can think of a scenario it may break that I haven't considered though.